src/api/generate-xlsx/map-application-to-XLSX/map-exporter/index.ts

```diff
@@ -59,7 +59,7 @@ export const mapBroker = (application: Application) => {
       xlsxRow(XLSX.FIELDS[BROKER_NAME], broker[BROKER_NAME]),
       xlsxRow(
         XLSX.FIELDS[ADDRESS_LINE_1],
-        `${broker[ADDRESS_LINE_1]} ${NEW_LINE} ${broker[TOWN]} ${NEW_LINE} ${broker[COUNTY]} ${NEW_LINE} ${broker[POSTCODE]}`,
+        `${broker[ADDRESS_LINE_1]} ${NEW_LINE}${broker[ADDRESS_LINE_2]} ${NEW_LINE}${broker[TOWN]} ${NEW_LINE}${broker[COUNTY]} ${NEW_LINE}${broker[POSTCODE]}`,
```

Comment:
is this better in a const since is a few lines

Reply:
Moved to a const 👍